packages/chart/src/CdcChart.tsx

```diff
@@ -566,6 +563,16 @@ export default function CdcChart({ configUrl, config: configObj, isEditor = fals
     }
   }
 
+  // function calculates the width of given text and its font-size
+  function getTextWidth(text: string, font: string) {
```

Comment:
Nice fn finding the text width on an SVG. It does look like it repeats the values over multiple times, ie. `94, 107, 114` but appears to get the desired result. Follows current code patterns passing it into context. Nice work!